### ISSUE TITLE
APP

### ISSUE THREAD

**Issue by eaedk** [Kyei-frank/Sentiment-Analysis-Project-Deep-Learning-#1]
Is your app working, please?
[APP LINK](https://huggingface.co/spaces/FKBaffour/Gradio_App_for_Sentiment_Analysis)

**Comment by Kyei-frank** [comment]
Yes please. It is working. It stopped for a while because of inactivity. But I have restarted the space so it is working now.
